providers/tests/system/microsoft/azure/example_adf_run_pipeline.py

```diff
@@ -22,12 +22,12 @@
 
 from airflow.models import DAG
 from airflow.models.xcom_arg import XComArg
+from airflow.providers.microsoft.azure.operators.data_factory import AzureDataFactoryRunPipelineOperator
+from airflow.providers.microsoft.azure.sensors.data_factory import AzureDataFactoryPipelineRunStatusSensor
 
 # Ignore missing args provided by default_args
 # mypy: disable-error-code="call-arg"
-from airflow.operators.empty import EmptyOperator
-from airflow.providers.microsoft.azure.operators.data_factory import AzureDataFactoryRunPipelineOperator
-from airflow.providers.microsoft.azure.sensors.data_factory import AzureDataFactoryPipelineRunStatusSensor
+from airflow.providers.standard.operators.empty import EmptyOperator
```

Comment:
Do we need to make it part of mypy disable?

Reply:
Hm just followed the general pattern and renamed occurences. Was added as part of #22974

Reply:
yep, but you moved other provider things out of this block so just want to confirm whether we could do that for empty operator

Reply:
I dont think we need to move it out, seems like its correct.

Reply:
sounds good